build.sh

```diff
@@ -210,6 +212,30 @@ cp tmp/lib/*/libuuid.so.1.3.0 rootfs/lib/libuuid.so.1
 # zlib1g components
 cp tmp/lib/*/libz.so.1  rootfs/lib/libz.so.1
 
+# wpa_supplicant components
+cp tmp/sbin/wpa_supplicant rootfs/sbin/wpa_supplicant
+cp -r tmp/etc/wpa_supplicant rootfs/etc/wpa_supplicant
+
+# firmware realtek
```

Comment:
This cleared up a misunderstanding on my part. I'm just learning this stuff on the way :)
I'm not even using the firmware-realtek package. I just ran through the installation without it. Why add more firmware packages? (ref first comment on update.sh)

Somethings come up, so next commit will have to wait for some more time.